packages/amplify-category-auth/provider-utils/awscloudformation/assets/adminAuth/admin-auth-app.js

```diff
@@ -175,6 +176,22 @@ app.get('/listUsers', async (req, res, next) => {
   }
 });
 
+app.get('/listGroups', async (req, res, next) => {
+  try {
+    let response;
+    if (req.query.token) {
+      response = await listGroups(req.query.limit || 25, req.query.token);
```

Comment:
Did we choose 25 here since that's the cognito user pool group limit? i believe they changed that recently to 10,000. https://docs.aws.amazon.com/cognito/latest/developerguide/limits.html

Reply:
I kept it at 25 because that was what all the other list operations are already set to.  A default of 25 results at a time seems reasonable to me.